### ISSUE TITLE
Feature request: Wake up free

### ISSUE THREAD

**Issue by hippyk** [MycroftAI/mycroft-core#2225]
I hope the assistant has been in a wake- on, continuously receiving voice commands. I try to remove the code waiting to wake up directly. But the stt server returns an unrecognizable error of audio. I guess maybe it's because the sending frequency is too fast. Is there any elegant way to achieve this?

**Comment by krisgesling** [comment]
Hi Hippyk,

Am I right in understanding you want to not have the wake up term and instead have Mycroft continuously listening and responding to anything you say?

This would require constant STT processing 24 hours a day which would be extremely expensive. It's certainly not something we are able to provide on Mycroft's free service. You would also have to add some logic to help it distinguish when the start and end of a command was. Currently Mycroft's STT limits the length of requests that can be sent to it, and will stop listening after a period of silence indicating that a shorter command has been provided. You might want to look into a streaming STT service like Google's. We recently added the foundations to support these but you would be paying per minute of audio transcribed which will add up very quickly.

Finally you would have to be comfortable with a device transmitting literally everything it hears to an external service, which I'm quite sure the majority of our Community would have a very big problem with.

So in short, this is not something we would implement as part of Mycroft Core, however the beauty of open source means you are absolutely free to fork the project if it's something you or others want to add.

**Comment by krisgesling** [comment]
Hi Hippyk, assuming this answered your question so I'm going to close this issue for now.
